SPECS/zstd/zstd.spec

```diff
@@ -1,96 +1,323 @@
-Summary:        Tools for zstd compression and decompression
+# enable asm implementations by default
+%bcond_without asm
+
+# enable .lz4 support by default
+%bcond_without lz4
+
+# enable .xz/.lzma support by default
+%bcond_without lzma
+
+# enable .gz support by default
+%bcond_without zlib
+
+# enable pzstd support by default
+%bcond_without pzstd
+
+# Our tooling/pipelines still use the old rpm version 4.17 which doesn't understand this,
+# and I can't figure out how to fix the tooling/pipelines, so just avoid using bcond for now
+%bcond_with gtest
+
 Name:           zstd
 Version:        1.5.5
-Release:        2%{?dist}
+Release:        5%{?dist}
+Summary:        Zstd compression library
+
 License:        BSD-3-Clause AND GPL-2.0-only
 Vendor:         Microsoft Corporation
-Distribution:   Mariner
-Group:          Applications/File
-URL:            https://facebook.github.io/zstd/
+Distribution:   Microsoft Azure Linux
+URL:            https://github.com/facebook/zstd
 Source0:        https://github.com/facebook/zstd/releases/download/v%{version}/%{name}-%{version}.tar.gz
-Requires:       %{name}-libs = %{version}-%{release}
+
+Patch1:         pzstd.1.patch
+
+BuildRequires:  make
+BuildRequires:  gcc %{?with_gtest:gtest-devel}
+%if %{with lz4}
+BuildRequires:  lz4-devel
+%endif
+%if %{with lzma}
+BuildRequires:  xz-devel
+%endif
+%if %{with pzstd}
+BuildRequires:  gcc-c++
+%endif
+%if %{with zlib}
+BuildRequires:  zlib-devel
+%endif
+BuildRequires:  execstack
 
 %description
-The zstd package contains programs for compressing and decompressing files
+Zstd, short for Zstandard, is a fast lossless compression algorithm,
+targeting real-time compression scenarios at zlib-level compression ratio.
 
-%package        devel
-Summary:        Header and development files for zstd
-Requires:       %{name} = %{version}-%{release}
-Provides:       lib%{name}-devel = %{version}-%{release}
+%package -n lib%{name}
```

Comment:
Why was the re-name needed? With the `Provides` below it seems harmless but seems unnecessary.

Reply:
reduce delta from inherited spec file